pinning.go

```diff
@@ -6,6 +6,8 @@ import (
 	"os"
 
 	"github.com/cilium/ebpf/internal"
+
+	"golang.org/x/sys/unix"
```

Comment:
See CI failure: https://ebpf.semaphoreci.com/jobs/e430af84-9549-413d-94d0-5c1b0d093d9d

`internal/unix` should be imported here, with wrappers and constants added to `types_other.go` and `types_linux.go`.

Reply:
Ah! I only ran the relevant TestMap* and TestProg* tests locally. Will fix it! Thank you.